chartsconfig/parser.go

```diff
@@ -94,11 +118,35 @@ func (cc *ChartsConfiguration) Validate() error {
 	return nil
 }
 
-func parseEpp(input []byte) ([]byte, error) {
-	out, err := epp.Parse(input)
-	if err != nil {
-		return nil, err
+func stubChart(b []byte, partialPath string) (*hchart.Chart, error) {
+	tpls := []*hchart.Template{{Data: []byte(b), Name: "charts.yml"}}
+
+	if partialPath != "" {
```

Comment:
If you provide a template first check the directory exists

Reply:
depending on wether we provide a default, this would need to change yes. But right now, there's no need, since `Walk` already returns an error when it doesn't exist, which is what you want, so you know you did something wrong, instead of silently ignoring the provided path.